docs/docs/topics/device-identity.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Device Identity
+description: >-
+  Learn about how WebAuthn is used to authenticate and authorize access using Device ID and state.
+---
+
+## Device Identity
+
+One of the core components of a zero-trust security model is device authentication. Think of it as a component of multi-factor authentication; In addition to "what you know" (password) and "who you are" (biometrics, face recognition, etc), device authentication covers "what you have" by confirming that the device you are using to access a system is trusted.
+
+Device authentication is becoming more widely discussed as more products begin to use or require it. For example, Windows 11 generated a lot of [news stories][verge-tpm] by [requiring TPM 2.0][win11-reqs].
```

Comment:
To be read as a follow-up to the previous paragraph:

```suggestion
While your identity provider validates the user, and tools like JWTs and mTLS validate the traffic, device authentication (through WebAuthn) validates the end user's device. This protects a trusted user from accessing sensitive data from a potentially unsafe device, like their personal computer or phone. Additionally, it can validate that the trusted device is in a safe state by confirming trusted/patched versions of the operating system and/or web browser.

Device authentication is becoming more widely discussed as more products begin to use or require it. For example, Windows 11 generated a lot of [news stories][verge-tpm] by [requiring TPM 2.0][win11-reqs].
```